nvflare/fuel/utils/class_utils.py

```diff
@@ -137,7 +121,14 @@ def get_module_name(self, class_name) -> Optional[str]:
         Returns:
             The module name if found.
         """
-        return self._class_table.get(class_name, None)
+        if class_name not in self._class_table:
+            return None
```

Comment:
Change to raise ConfigError with text:
f"Cannot find class '{class_name}'. Please check its spelling. If the spelling is correct, specify the class using its full path."

Reply:
We already have the proper error message when calling this when return None. But we can change to raise this ConfigError here as well.